### PR TITLE
2021-03-12-deeamon128-practice-html

### PR DESCRIPTION
**Pull request by deeamon128**

### CODE REVIEW

**Review by swoopstyle — APPROVED** [review]
💯 💯

**Review by Constantin-Matei — APPROVED** [review]
Nice!

File: web/deeamon128/html/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>hello hello</title>
+</head>
+<body>
+  <h1>test test</h1> 
+  <p> test1</p>  
+  <p>first try lalala</p>  
+</body>
+</html>
```
Comment (QuintyHH) [inline web/deeamon128/html/index.html:14]: Please leave an empty line at the end of the file :) 💯

**Review by QuintyHH — APPROVED** [review]

**Review by RalucaRbc — APPROVED** [review]
Great, well done 👍

**Review by marianzburlea — APPROVED** [review]
💯

**Comment by adelacecilia** [comment]
Cute 🥳

**Pull request merged**